PresentScreenings/Controllers/ScreeningDialogController.cs

```diff
@@ -112,8 +112,15 @@ public override void GoToScreening(Screening screening)
         #region Private Methods
         private void SetControlValues()
         {
-            _control.Selected = true;
-            _filmInfoButton.Action = new ObjCRuntime.Selector("ShowFilmInfo:");
+            // Set the Info Button image as applicable.
+            var imageByAvailability = new Dictionary<bool, NSImage> { };
+            imageByAvailability[true] = _filmInfoButton.Image;
+            imageByAvailability[false] = _filmInfoButton.AlternateImage;
+            var isAvailable = ViewController.FilmInfoIsAvailable(_screening.Film);
+            _filmInfoButton.Image = imageByAvailability[isAvailable];
```

Comment:
Please consider if a local dictionary isn't overkill.

Reply:
Kept it, I like a `dict` much more than an `if` `then` `else`.
Dit add some comments though.